src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -208,7 +208,7 @@ public Integer applyVacationRuleToEmployee(Employee employee, String vacationYea
             if (currentDate.getMonthValue() == Month.OCTOBER.getValue() && startDate.isBefore(LocalDate.of(currentDate.getYear(), Month.JULY, 1))) {
                 int yearDiff = startDate.until(currentDate).getYears();
                 int vacationDays = activeContract.get().getSeniority().getVacationDays();
-                return yearDiff >= 2 ? 15 : vacationDays;
+                return yearDiff >= 5 ? 20 : (yearDiff >= 2 ? 15 : vacationDays);
```

Comment:
This is ok, but let's extract this logic to a new method, similar to `vacationDaysPerWorkDay`